src/docs/flex-cli/manage-search-schemas-with-flex-cli/index.md

```diff
@@ -111,9 +120,19 @@ flex-cli search set --key category --type enum --scope public -m my-test-marketp
 flex-cli search set --key amenities --type multi-enum --scope public -m my-test-marketplace
 ```
 
+Note that we don't need to supply `--schema-for` option when we add schema for `listing` as `listing` is the default.
```

Comment:
I remember some time ago a rule of thumb to avoid "we" and "you" distinctions in our docs or communication. Something about it creating a polarity in the reader's mind. I personally have been avoiding that, but not sure it's always that easy. What do you think, @SariSaar?

Reply:
That's an interesting point. Should I change it to:

Note that `--schema-for` option is not needed when adding schema for `listing` as `listing` is the default.

?